subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ProtobufPluginSmokeTest.groovy

```diff
@@ -61,19 +60,14 @@ class ProtobufPluginSmokeTest extends AbstractPluginValidatingSmokeTest {
         """
 
         when:
-        def result = runner('compileJava').forwardOutput()
-            .expectDeprecationWarning(deprecationOfFileTreeForEmptySources("sourceFiles"), "https://github.com/google/protobuf-gradle-plugin/pull/530")
-            .build()
+        def result = runner('compileJava').forwardOutput().build()
 
         then:
         result.task(":generateProto").outcome == SUCCESS
         result.task(":compileJava").outcome == SUCCESS
 
         when:
-        result = runner('compileJava')
-            .forwardOutput()
-            .expectDeprecationWarning(deprecationOfFileTreeForEmptySources("sourceFiles"), "https://github.com/google/protobuf-gradle-plugin/pull/530")
-            .build()
+        result = runner('compileJava').forwardOutput().build()
```

Comment:
Nit: shall we keep the original formatting of separate lines? I think it makes sense to do so in tracking what's changed (and future deprecations, too).

Reply:
I reverted this to the state before I added the `expectDeprecationWarning` here. Everything was on one line. I don't feel it helps much having everything on a separate line.